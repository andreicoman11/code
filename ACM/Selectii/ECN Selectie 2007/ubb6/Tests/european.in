28
4
1524 1520 1609 1435
3
1000 1520 1600
6
1000 2000 3000 4000 1500 2500
4
1000 1050 2000 1950
4
1000 1400 4000 1600
4
2940 1040 1010 3950
5
1110 2340 1230 1320 2550
5
3000 2000 1950 1050 1000
5
1950 1010 2060 2050 1050
6
2000 2050 1000 1050 1020 2030
6
4060 1000 1010 1020 1030 2010
6
4060 1000 1010 1020 1030 3030
6
2020 1000 1050 1020 1110 3130
7
1000 1001 2498 1500 2500 3998 4999
7
4999 3998 2500 1500 2498 1001 1000
8
2999 3000 4000 4001 4003 4004 1001 1004
1
2909
1
1598
1
2285
2
3662 3287
2
4659 3171
2
3805 4736
3
4802 3174 1002
3
2271 1820 3799
3
3420 1059 4613
4
2213 4954 1529 1934
4
2909 3516 1110 3866
4
3462 3050 1360 1861
