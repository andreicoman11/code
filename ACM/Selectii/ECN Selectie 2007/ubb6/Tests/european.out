Scenario #1
4: 0 1520 1609 3044

Scenario #2
4: 0 1000 1520 1600

Scenario #3
5: 0 1500 3000 4000 5000

Scenario #4
4: 0 1000 1050 3000

Scenario #5
4: 0 1400 2400 4000

Scenario #6
4: 0 1010 2050 3950

Scenario #7
4: 0 210 1320 2550

Scenario #8
4: 0 1950 2000 3000

Scenario #9
5: 0 1010 1950 2050 2060

Scenario #10
5: 0 1020 2000 2050 3050

Scenario #11
5: 0 1000 2010 3030 4060

Scenario #12
5: 0 1000 2010 3030 4060

Scenario #13
5: 0 1110 2110 2160 3130

Scenario #14
5: 0 2498 3998 4998 4999

Scenario #15
5: 0 1500 2500 3998 4999

Scenario #16
5: 0 3 1004 4003 4004

Scenario #17
2: 0 2909

Scenario #18
2: 0 1598

Scenario #19
2: 0 2285

Scenario #20
3: 0 3287 3662

Scenario #21
3: 0 3171 4659

Scenario #22
3: 0 3805 4736

Scenario #23
4: 0 1002 3174 4802

Scenario #24
4: 0 1820 2271 3799

Scenario #25
4: 0 1059 3420 4613

Scenario #26
5: 0 1529 1934 2213 4954

Scenario #27
5: 0 1110 2909 3516 3866

Scenario #28
5: 0 1360 1861 3050 3462

